setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(name='tap-facebook',
-      version='1.7.0',
+      version='1.7.1',
```

Comment:
In the future, please leave this to us to bump.

Reply:
:+1:  will do!